alsoHeresTheInterviewQuestionIHad/alsoHeresTheInterviewQuestionIHad.js

```diff
@@ -0,0 +1,31 @@
+const fs = require("fs");
+const streetArray = fs.readFileSync("data.txt", "utf-8").split(",");
+let answer = "";
+let tallestBuilding = Math.max(...streetArray);
+let numberOfBuildings = streetArray.length;
+let tallestBuildingSoFar = 0;
+let bestArea = tallestBuilding * numberOfBuildings;
+let bannerOne = [tallestBuilding, numberOfBuildings];
+let bannerTwo = [tallestBuilding, numberOfBuildings];
+let bestBannerOne = [tallestBuilding, numberOfBuildings];
+let bestBannerTwo = [tallestBuilding, numberOfBuildings];

```

Comment:
nit: I'd personally prefer if there were some newlines in here to organize these a bit. It's personal preference though.

Reply:
Hmmm, this does feel like will be very subjective, but l will try to read the tea leaves of what those who come after might prefer.

Reply:
Marking this resolved as there wasn't a fresh complaint.  Prepared for that to change, as I might adjust these inits as per lower comments.